src/ui/buttons/props.js

```diff
@@ -310,11 +310,10 @@ export function normalizeButtonStyle(props : ?ButtonPropsInputs, style : ButtonS
         throw new Error(`Expected ${ fundingSource || FUNDING.PAYPAL } to be eligible`);
     }
 
-    style.color = style.color ? style.color : fundingConfig.colors[0];
     const {
         label,
         layout = fundingSource ? BUTTON_LAYOUT.HORIZONTAL : fundingConfig.layouts[0],
-        color = fundingConfig.colors[0],
+        color = style.color || fundingConfig.colors[0],
```

Comment:
`style.color || fundingConfig.colors[0]` should cover an empty string right? Why do we need to cover a whitespace string? Whitespace should result in a validation error being thrown.

Reply:
I see what you mean about whitespace. Empty string is still a problem though. `style.color || fundingConfig.colors[0]` does not cover empty string since it is not undefined and the check for valid color values doesn't catch it because it is falsy. Adding `style.color ||` is unnecessary since color would already be set as style.color unless it was undefined. We could modify line 113 to update empty string to `undefined` or update the color check to also throw and error for empty strings. Maybe just switch it to something like
```
style.color = style.color ? style.color : undefined;
```

Reply:
By doing this:
```js
style.color = style.color ? style.color : undefined;
```
You would go back to mutating the original style object.

Also this expression `style.color ? style.color : fundingConfig.layouts[0];` and this other expression `style.color || fundingConfig.layouts[0];`  would give the same result. If `style.color` is a falsy value such as `0`, `''`, `null`, `undefined`, etc, it will take the second part of the statement, which would be `fundingConfig.layouts[0]`.

As @bluepnume stated, covering empty strings, null, etc. should be enough. But of course I'm open to hear other opinions!

Reply:
Hi @PapaniiPPCorp thanks for your feedback and helping identify this bug. I think @amyegan solved this in #1573 and included some unit tests to prevent regressions. The `style` prop should no longer be mutated. 

We are in the process of deploying this change to prod. It will be in the 5.0.211 release and should be showing up in the [sdk script](https://www.paypal.com/sdk/js?client-id=test) in the next hour. Please test 5.0.211 on your end and if everything looks good then we can close out this PR.

Reply:
Great, I'll close this PR then. Thanks!